asm/canonical-service/controller.yaml

```diff
@@ -327,7 +328,7 @@ spec:
         - --enable-leader-election
         command:
         - /manager
-        image: gke.gcr.io/asm/canonical-service-controller:1.10.3-asm.16 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub"}
+        image: gke.gcr.io/asm/canonical-service-controller:1.22.3-asm.1 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.canonicalServiceHub"}
```

Comment:
We don't have this version of image in the image registry. Can you make sure the image exists first?

https://pantheon.corp.google.com/gcr/images/gke-release/global/asm/canonical-service-controller?e=-13802955&inv=1&invt=AbXdYQ&jsmode=o&mods=prod&project=gke-release

Reply:
Image is available now